### DISCUSSION TITLE
vector3 곱 연산이 안됩니다..

### DISCUSSION THREAD

**Discussion by ssm00** [naverz/zepeto-studio-kor#711] [5. Q&A]
안녕하세요 다름이 아니라 벡터를 새로 계산해서 위치를 조정하고 싶은데 자꾸 컴파일이 안되서 질문드려요
이전에 빨간줄로 에러생기는거는 해결을 했습니다.
근데 오류는 안생겨도 백터*number 연산을 하면 컴파일이 안되더라고요 빨간줄로 에러는 안뜨는데 
사진2번처럼 컴파일이 안되네요

일단 제가 돌렸을때는 백터끼리 합은 되고 백터와 상수의 곱이 안되는거 같아요

여기서 저렇게 백터 * 상수만 밑에 사진처럼 컴파일 에러가 뜨더라고요
빨간줄 에러는 안생기는데 백터*상수 한줄 추가하면 컴파일이 안되더라고요..
백터 * 상수 라인 한 줄 지우면 잘 돌아가고요
<img width="446" alt="3" src="https://user-images.githubusercontent.com/97657265/167260555-713c47b0-2328-4321-93cb-5769b4cb64b8.PNG">

<img width="559" alt="2" src="https://user-images.githubusercontent.com/97657265/167260031-4d2a541f-818a-4ae1-8a68-5b6f2628256a.PNG">
그래서 코드자체가 안돌아 가는거같은데 혹시 백터연산 쓰시는분들은 곱은 어떻게 사용하시는지 알 수 있을까요?
혹은 제가 뭔가를 잘못하고 있나요??

추가로 혹시 캐릭터위치를 바꾸고 싶으면 moveToPosition메소드를 사용하면 되는걸까요?
아님 그냥 move를 사용하는게 나을까요?

**Comment by smilejsu82** [comment]
함수를 하나 만들어서 해보시는건 어떨까요?
![image](https://user-images.githubusercontent.com/50188795/167339107-55e86ab3-d74c-4391-a106-01dc9cf4f53e.png)
![image](https://user-images.githubusercontent.com/50188795/167339124-d7766fd4-79d9-4fe0-a10b-ebf830cbca25.png)

**Reply by ssm00** [reply]
감사합니다. 연산이 저만 안되는건 아닌거죠?? 왜 곱만 컴파일이 안되는지 잘 모르겠네요 혹시 백터곱 계산 사용하실때 항상 메소드 만들어서 사용하셨나요???

**Reply by ted002** [reply]
아 그 왜그러는지는 모르겠는데 제페토 타입스크립트에서는 그게 안되더라구요...
굳이 만드실필요 없이 Vector3.op_Multiply(v1,v2)쓰시면 됩니다
더하는것도 비슷하게 있습니다

**Reply by smilejsu82** [reply]
> 아 그 왜그러는지는 모르겠는데 제페토 타입스크립트에서는 그게 안되더라구요...
> 
> 굳이 만드실필요 없이 Vector3.op_Multiply(v1,v2)쓰시면 됩니다
> 
> 더하는것도 비슷하게 있습니다

벡터 * 실수도 되나요?

**Reply by smilejsu82** [reply]
오호 이런게 있었군요 감사합니다 ~ 
![image](https://user-images.githubusercontent.com/50188795/167417753-2a74a624-389d-457f-a139-acd6894d5c3f.png)

**Reply by ssm00** [reply]
감사합니다~
